packages/visualizations/package.json

```diff
@@ -82,7 +82,7 @@
         "rollup-plugin-svelte": "^6.1.0",
         "rollup-plugin-terser": "^7.0.2",
         "rollup-plugin-visualizer": "^4.2.0",
-        "svelte": "^3.43.2",
+        "svelte": "^3.59.2",
```

Comment:
Not blocking for this PR, but why was it needed? We probably should update to 4 (and 5 quickly) anyways.

Reply:
Need it to use CSS container queries. Support starts from v3.58.0